Source/panels/charpanel.cpp

```diff
@@ -0,0 +1,315 @@
+#include "panels/charpanel.hpp"
+
+#include <fmt/format.h>
+#include <string>
+
+#include "DiabloUI/art.h"
+#include "DiabloUI/art_draw.h"
+
+#include "control.h"
+#include "engine/render/cel_render.hpp"
+#include "engine/render/text_render.hpp"
+#include "utils/language.h"
+
+namespace devilution {
+
+struct colorAndText {
+	UiFlags color;
+	std::string text;
+};
+
+struct panelEntry {

```

Comment:
Code style says capital first letter.
```suggestion
struct PanelEntry {
```